dapp/src/components/strapi/seo.js

```diff
@@ -0,0 +1,93 @@
+import Head from 'next/head'
+import { useContext } from 'react'
+import { GlobalContext } from '../../../pages/_app'
+import { getStrapiMedia } from '../../../lib/media'
+
+const Seo = ({ seo }) => {
+  const { defaultSeo, siteName } = useContext(GlobalContext)
+  const seoWithDefaults = {
+    ...defaultSeo,
+    ...seo,
+  }
+  const fullSeo = {
+    ...seoWithDefaults,
+    // Add title suffix
+    metaTitle: seoWithDefaults.article
+      ? `${seoWithDefaults.metaTitle} | ${siteName}`
+      : `${seoWithDefaults.metaTitle}`,
+    // Get full image URL
+    shareImage: getStrapiMedia(seoWithDefaults.shareImage),
+  }
+
+  return (
+    <Head>
+      {fullSeo.metaTitle && <title>{fullSeo.metaTitle}</title>}
```

Comment:
if `metaTitle` isn't set, let's set it to `Origin Dollar` or something. Empty title/URL looks ugly

Reply:
done